frontend/packages/ceph-storage-plugin/src/components/disk-inventory/ocs-disks-list.tsx

```diff
@@ -181,9 +184,8 @@ const OCSDisksList: React.FC<TableProps> = React.memo((props) => {
 
     if (tiLoaded && !tiLoadError && tiData.length) {
       const newData: OCSDiskList = tiData.reduce((data, ti) => {
-        const disk = ti.metadata.annotations?.disk;
-        const osd = ti.metadata.annotations?.osd;
-        if (osd && disk) {
+        const { disk, node, osd } = getAnnotations(ti) || {};
+        if (osd && disk && node === nodeName && ocsState.metricsMap?.[disk]?.osd === osd) {
```

Comment:
If we know the nodeName we can watch only that node using node name in https://github.com/openshift/console/pull/8864/files#diff-2fe4b3881950e5603a824223a9a6a00e3955713e1ee8d6ece63aff914590257aR142,